src/flightlog_fielddefs.js

```diff
@@ -526,6 +526,10 @@ export function adjustFieldDefsList(firmwareType, firmwareVersion) {
     if (semver.lt(firmwareVersion, "4.6.0")) {
       DEBUG_MODE.splice(DEBUG_MODE.indexOf("D_MAX"), 1, "D_MIN");
     }
+    if (semver.gte(firmwareVersion, "4.7.0")) {
```

Comment:
4.7?

Reply:
i see, i suppose this keep compatibility for earlier version of 4.6.

Reply:
No you are right - used to 1.47 being 4.6 :)